tests/ui/parser/issues/diagnostics-parenthesized-type-arguments-issue-120892-3.stderr

```diff
@@ -0,0 +1,16 @@
+error: expected type, found `1`
+  --> $DIR/diagnostics-parenthesized-type-arguments-issue-120892-3.rs:5:19
+   |
+LL |             Foo::(1, 2) => {},
+   |                ---^ expected type
+   |                |
+   |                while parsing this list of parenthesized type arguments starting here
```

Comment:
```suggestion
   |                while parsing this parenthesized list of type arguments starting here
```

I know that I provided the original wording in the corresponding issue but the suggested phrasing is more correct.

Reply:
Applied.